src/js/media-chrome-listbox.js

```diff
@@ -7,6 +7,26 @@ template.innerHTML = `
 <style>
   :host ul {
     list-style: none;
+    display: flex;
```

Comment:
Thanks for adding default styles!
Any thoughts on making this inline-flex so it just fits the content?

Reply:
that probably makes sense. I'll take a look and tweak styles as appropriate.

Reply:
updated